assets/src/application/ui/calendars/dateDisplay/MediumCalendarDate/style.scss

```diff
@@ -45,7 +45,7 @@
 			letter-spacing: var(--ee-letter-spacing-font-size-micro);
 			line-height: var(--ee-font-size-default);
 			left: var(--ee-padding-nano);
-			margin: var(--ee-margin-nano) 0 var(--ee-margin-tiny);
```

Comment:
this was pushing footer content too much:

![image](https://user-images.githubusercontent.com/1477453/79157099-a69ff780-7ddc-11ea-89fa-6208ea1ba0a0.png)

Reply:
I'll have to tweak a few styles cuz that's looking a little cramped... 
think i might have the card height too low